src/main/java/frc/robot/commands/DriveCommand.java

```diff
@@ -5,36 +5,56 @@
 import com.ctre.phoenix6.swerve.SwerveModule.DriveRequestType;
 import com.ctre.phoenix6.swerve.SwerveRequest;
 import dev.doglog.DogLog;
+import edu.wpi.first.math.MathUtil;
 import edu.wpi.first.math.controller.PIDController;
 import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
 import frc.robot.generated.TunerConstants;
 import frc.robot.subsystems.CommandSwerveDrivetrain;
 
 public class DriveCommand extends Command {
-  private double MaxSpeed =
-      TunerConstants.kSpeedAt12Volts.in(MetersPerSecond); // kSpeedAt12VoltsMps desired top speed
-  private double MaxAngularRate =
-      3.5 * Math.PI; // 3/4 of a rotation per second max angular velocity
+  private static final double PID_MAX = 0.35;
 
-  public static final double PID_MAX = 0.35;
-
-  private PIDController PID;
-  private CommandSwerveDrivetrain drivetrain;
-  private CommandXboxController driverController;
+  private final CommandSwerveDrivetrain drivetrain;
+  private final CommandXboxController driverController;
+  private final PIDController PID;
   private Pose2d currPose;
+
+  public boolean isBackCoralStation = false;
   public boolean isSlow = false;
+
+  private double MaxSpeed = TunerConstants.kSpeedAt12Volts.in(MetersPerSecond);
+  private double MaxAngularRate = 3.5 * Math.PI;
+
+  private final double REDLEFTSTATIONANGLE = 126;
+  private final double REDRIGHTSTATIONANGLE = -126;
+  private final double BLUELEFTSTATIONANGLE = -54;
+  private final double BLUERIGHTSTATIONANGLE = 54;
```

Comment:
Should add underscores in between the words to make it easier to read.

I.e, (only using the first one for brevity)

`REDLEFTSTATIONANGLE` --> `RED_LEFT_STATION_ANGLE`